### DISCUSSION TITLE
posts/national-cycle/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#32] [Comments]
# posts/national-cycle/

工作后真切感受到省外小伙伴的返乡压力，即便蹲着高铁票放出的点，还是没在一秒内抢到票。通过「买短乘长」或「买长乘短」坐上了高铁，就会觉得自己是返乡幸运儿。抢不到票的小伙伴好不容易拼了车，又被堵在高速上。
[&hellip;] 俺则是在爸妈的不解和奶奶的担心中踏上了骑行返乡路。骑行回家大约 62 公里，背上电脑，带上能量胶和电解质水1，早上七点啃上两个包子就出发。大约 3 小时到家，主要卡在市区的红绿 &hellip;

https://shitao5.org/posts/national-cycle/

**Comment by earfanfan** [comment]
哇，3小时62公里好厉害啊，这是怎么做到的，买专业的骑行装备就可以吗？我之前在武汉的城市绿道骑行，用共享单车，3小时只能骑20几公里。你的事迹给了我一点鼓舞，之前我从来没想象过骑行的上限会超过30公里，现在我决定多去试试。

**Reply by Shitao5** [reply]
3小时62公里就是回家心切一直蹬，我猜你3小时骑20几公里是中途下来拍照了哈哈。专业的骑行装备（公路车、平把公路车）比起共享单车会轻松很多，主要是通过变档位保持合适的踏频，就像自动挡的汽车会保持转速在1500左右一样。而共享单车要是想骑快些，就像手动挡汽车挂了1档猛踩油门。当然啦，肯定不是买专业骑行装备就可以的，大腿（发动机）决定了上限，多骑一骑就没问题！
